mitodl/odl-video-service
# make video title required

#### What are the relevant tickets?

closes #209 

#### What's this PR do?

this makes `title` a required field in the `Video` model. We already have stuff on the frontend for showing serializer validation errors on our forms, so we get the frontend UI behavior we want for free.

I also added a test for the validation message for the video title and also one for the collection title (testing that a blank title can't be saved without raising an exception).

#### How should this be manually tested?

Edit a video, and try to save it with a blank title. You should get a little 'hey, you need this' type validation error, and you shouldn't be able to save it.
